### ISSUE TITLE
OverrideFont= Does not work!?

### ISSUE THREAD

**Issue by K14Mua** [bbepis/XUnity.AutoTranslator#457]
How should the font substitution function work? What do you need to write there that the font has been replaced?

I wrote
OverrideFont=Arial
OverrideFont=Calibri
OverrideFont=Candara
OverrideFont=Tahoma
OverrideFont=Verdana
OverrideFont=Segoe UI
OverrideFont=Georgia

But nothing changed in the game. The native font is still displayed. Are these fonts not suitable? How to use this option? The readme doesn't say anything useful about this...

**Comment by Fialonzy** [comment]
Similar problem
Do you display text using UGUI?

Here's what the documentation says:

> OverrideFont= ;Overrides the fonts used for texts when updating text components. NOTE: Only works for UGUI

**Comment by K14Mua** [comment]
> Similar problem Do you display text using UGUI?
> 
> Here's what the documentation says:
> 
> > OverrideFont= ;Overrides the fonts used for texts when updating text components. NOTE: Only works for UGUI

How can you understand what text is displayed through the UGUI? Could it be that there is no UGUI text in the game at all?

**Comment by Fialonzy** [comment]
I was going through the settings.
I turned off different types of text one by one and determined which one was responsible for the translation.
```
[TextFrameworks]
EnableUGUI=True                  ;Enable or disable UGUI translation
EnableNGUI=True                  ;Enable or disable NGUI translation
EnableTextMeshPro=True           ;Enable or disable TextMeshPro translation
EnableTextMesh=False             ;Enable or disable TextMesh translation
EnableIMGUI=False                ;Enable or disable IMGUI translation
```

I set everything to False except one and looked to see if there was a translation

**Comment by K14Mua** [comment]
> I was going through the settings. I turned off different types of text one by one and determined which one was responsible for the translation.
> 
> ```ini
> [TextFrameworks]
> EnableUGUI=True                  ;Enable or disable UGUI translation
> EnableNGUI=True                  ;Enable or disable NGUI translation
> EnableTextMeshPro=True           ;Enable or disable TextMeshPro translation
> EnableTextMesh=False             ;Enable or disable TextMesh translation
> EnableIMGUI=False                ;Enable or disable IMGUI translation
> ```
> 
> I set everything to False except one and looked to see if there was a translation

Well, everything is clear, I did it like this
```
[TextFrameworks]
EnableIMGUI=False
EnableUGUI=True
EnableNGUI=False
EnableTextMeshPro=False
EnableTextMesh=False
EnableFairyGUI=False
```
And the translation completely disappeared from the game.

So there's no way I can play around with the translation?

**Comment by K14Mua** [comment]
All the text is there `EnableTextMeshPro=True`

**Comment by Fialonzy** [comment]
I'm also trying to solve a font problem. There are no results yet.

If the translation appears, then the game is using TextMeshPro.
There is a solution for it, but personally it doesn’t work for me.

I'm currently looking for what parameter can be set for OverrideFontTextMeshPro and FallbackFontTextMeshPro...

**Comment by Fialonzy** [comment]
#57 
This point is discussed here, read it.
If you understand how to do it, then write, I will be grateful

**Comment by K14Mua** [comment]
A very strange moment. Some, I repeat, SOME letters turn into squares after translation. That being said, when I disabled EnableUGUI, I cleared the translations folder. And the program translated the text again. And in this new text there are no squares.

What's the matter?

Old translation
![image](https://github.com/bbepis/XUnity.AutoTranslator/assets/24901436/139809c5-24aa-4195-aedd-8708adb906d7)

New translation
![image](https://github.com/bbepis/XUnity.AutoTranslator/assets/24901436/6e6c3e04-36b1-40f3-a7ce-e1415f6ca125)
